api/src/passage/models.py

```diff
@@ -83,3 +82,42 @@ class PassageHourAggregation(models.Model):
     electric = models.IntegerField(null=True)
     toegestane_maximum_massa_voertuig = models.TextField()
     count = models.IntegerField()
+
+
+class ZwaarVerkeerHelperTable(models.Model):
+    camera_naam = models.CharField(max_length=255, db_index=True)
+    rijrichting = models.IntegerField(null=True, blank=True, db_index=True)
+    camera_kijkrichting = models.FloatField(null=True, blank=True, db_index=True)
+
+    order_kaart = models.IntegerField(null=True, blank=True)     # in sheet: volgorde
+    order_naam = models.CharField(max_length=255, null=True, blank=True)      # in sheet: straatnaam
+    cordon = models.CharField(max_length=255, db_index=True, null=True, blank=True)
+    richting = models.CharField(max_length=10, null=True, blank=True)
+    location = models.PointField(srid=4326, null=True, blank=True)
+    geom = models.CharField(max_length=255, null=True, blank=True)
+    azimuth = models.FloatField(null=True, blank=True)
+
+
+class ZoneZwaarVerkeer(models.Model):
```

Comment:
Ik vind deze naam een beetje generiek. Het huidige model heet `PassageHourAggregation` - daaruit is het duidelijk dat het om geaggregeerde data gaat. Misschien kunnen we ook zoiets bedenken voor dit model, ook om het enigszins consistent te maken.

Reply:
Aangepast naar `HeavyTrafficHourAggregation`.